nss-day-cohort-51/nss-day-cohort-51.github.io
# Ss navbar fix 141

# Description
Created all new navbart

## Type of change


# Testing Instructions
Fetch the branch and do npm start. Resize your screen to and check the following sizes to make sure navbar looks like it should and also does not rerender when links are clicked: 360px, 768px, & 992px (or larger).
Also make sure links scroll to correct position on page on both mobile and desktop sizes.

# Checklist:
